roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -10,6 +10,16 @@
 import re
 from collections import defaultdict
 
+try:
+
+    import requests
```

Comment:
I don't think we need this kind of construction. It's better to fail fast if the environment is not as expected.

Reply:
The code will fail a bit later with the same behavior.
I did this to avoid adding `requests` as requirement for unit testing knowing that this is not used and could potentially been replaced by another library with the same behavior

Reply:
The unit-tests won't run in the same environment, it's a different thing. We've got the controller on the container that we use to run `ansible-test-splitter`. Right now, it's basically a Fedora with the ansible-core package.

If we were to upgrade the container with something that doesn't include requests, we would prefer a quick failure of the job. This instead of discovering 1 month later that in some specific cases the job fail. It's often better to fail fast.

Reply:
I remove the `try...except` block, unit tests are failing now